docs/admin/install/steps/install-configure.rst

```diff
@@ -20,8 +20,12 @@ Configuring Weblate
 
         weblate migrate
 
-#. Create an account for the administrator user and copy its password
-   to the clipboard, and also save it for later use:
+#.  Create an account for the administrator user. This command will generate an admin password for the first use.
+    Copy the password to the clipboard, and also save it for later use:
+
+    If you previously missed/lost the admin password, you can generate a new one with the following command:
+
+        weblate createadmin --update
```

Comment:
```suggestion
    .. hint::
    
       If you previously missed/lost the admin password, you can generate a new one with the following command:
       
       .. code-block:: sh

           weblate createadmin --update
```

...and possibly move it below the code block so that the `weblate createadmin` command is the first one to notice.